README.md

```diff
@@ -56,10 +61,41 @@ op.attempt(next => {
 
 ```
 
+### `Async/await` Support
+
+Retryme now supports `async/await`. It accepts three arguments:
+
+- `retry`: The configuration for create a new [retryme instance][retryme-instance].
+- `attempt`: The real `async` function which will be retried with `await`.
+- `ignore`: The `ignore` function also for setup [retryme instance][retryme-instance] to terminate retries.
+
+A simple example of how to use it is as follows:
+
+```js
+const awaitRetryme = require('retryme/await');
```

Comment:
part of me is considering a different api, but I have not experimented with its feasibility but it seems like it could work out, returning an `async` function from the `retryme` instance. We would still use the same `retryme` api like.
```js
const retry = new Retryme(opts, ignore);

// but then we could return an async function which handles the retry attempts 
// internally.
try {
  const result = await retry.await(() => someAsyncFunc(val))
} catch (err) {
  // handle error
  console.error(err);
}
```